hub/scripts.js

```diff
@@ -234,10 +245,16 @@
         // add new meta tag
         $frag.appendChild(createTag('meta', m));
       }
+      // check if property can be found in the sharedMetaImage that contains the og:image
```

Comment:
The whole block is not needed here since you don't want to override authored images with the default value

Reply:
removed